tagging.py

```
@@ -1,16 +1,503 @@
+    
+
+
+
+class Lexer:
+    
+    def __init__( self, string: str, allow_issue_start_with_letter: bool = False) -> None:
+        self.input: str = string
+        self.state: LexerFunc | None = None
+        self.pos: int = -1
+        self.start: int = 0
+        self.lastPos: int = 0 
```

Flake8 (7.2.0)

[warning] trailing whitespace

(W291)